Gems/Multiplayer/Code/Source/MultiplayerSystemComponent.cpp

```diff
@@ -1220,21 +1285,21 @@ namespace Multiplayer
             AZStd::vector<NetBindComponent*> gatheredEntities;
             AZ::Interface<AzFramework::IVisibilitySystem>::Get()->GetDefaultVisibilityScene()->Enumerate(viewFrustum,
                 [this, &gatheredEntities](const AzFramework::IVisibilityScene::NodeData& nodeData)
-            {
```

Comment:
This is a good example, it's really hard to tell if you actually touched anything in here without manually diffing almost line by line.

Reply:
Reverted whitespace changes. It should be good now.